### PR TITLE
Katello 4.0 RC documentation updates

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ defaults:
       path: "plugins/katello"
     values:
       layout: "plugins/katello/documentation"
-      latest: "3.18"
+      latest: "4.0"
```
Comment (jturel) [inline _config.yml]: Don't bump latest until 4.0 is GA :)

Comment (jjeffers) [reply _config.yml]: @jturel should we still merge this PR, assuming we don't bump latest as above? Or should we wait until GA to merge?

Comment (jturel) [reply _config.yml]: We can merge this as soon as RC1 hits the repos. Recommend sharing the draft release notes in IRC so others can have a look (kind of a tall order for one person to generate them imo)

Comment (jjeffers) [reply _config.yml]: Good idea!

**Comment by jjeffers** [comment]
Closing in favor of new docs site.

**Pull request closed without merging**